### ISSUE TITLE
testing installation only works halfway

### ISSUE THREAD

**Issue by Funky-Fresh** [mrklein/openfoam-os-x#72]
Hello, 

I've installed Openfoam-8 on macOS Catalina 10.15.7 using your instructions and when testing the cavity tutorial it seems working. However staying in the cavity directory and executing 

`
$ cat > system/decomposeParDict
FoamFile
{
    version     2.0;
    format      ascii;
    class       dictionary;
    location    "system";
    object      decomposeParDict;
}

numberOfSubdomains 4;

method          scotch;
^D
$ decomposePar
$ mpiexec -np 4 icoFoam -parallel
`

it says that file/directory system/decomposeParDict is not found. The same problem occurs when only running the _blockMesh_ command for the cavity tutorial.

Do you know why it does not find this directory?

I also tested it with the pitzdaily tutorial, which works fine.

**Comment by mrklein** [comment]
Hi,

Could elaborate on "staying in the cavity directory and executing"? Did you execute this command par command, or just copied-and-pasted the whole block to the terminal? Is there `system/decomposeParDict` file?

**Comment by Funky-Fresh** [comment]
I executed command of the cavity tutorial first and then, still in the cavity folder `$ cd cavity`. Then I executed the first command block 

```
$ cat > system/decomposeParDict

FoamFile
{
version 2.0;
format ascii;
class dictionary;
location "system";
object decomposeParDict;
}

numberOfSubdomains 4;

method scotch;
^D
```

with the result: no `system/decomposeParDict` and the other commands `not found`. The same happened when running ´blockMesh´ (no system/decomposeParDict`).

Until now I also looked up the tutorial and tried it with `cp -r $FOAM_TUTORIALS/incompressible/icoFoam/cavity/cavity . `
instead of `cp -r $FOAM_TUTORIALS/incompressible/icoFoam/cavity . ` 

This seems to add the `system/decomposeParDict` after running `foamRunTutorials` (also `blockMesh`is working now)

But afterwards, the command `cat > system/decomposeParDict....` just result in nothing. It shows no errors, but it also does not seem to calculate anything and I have to always exit with control-c.
